services/common/src/constants/actionTypes.ts

```diff
@@ -9,6 +9,7 @@ export const CLEAR = "CLEAR";
 export const OPEN_MODAL = "OPEN_MODAL";
 export const CLOSE_MODAL = "CLOSE_MODAL";
 export const UPDATE_MODAL_TITLE = "UPDATE_MODAL_TITLE";
+export const STORE_IS_LOADING = "STORE_IS_LOADING";
```

Comment:
The name for this is a bit confusing. I see that it is used to check if form is loading.

Reply:
👍 
Will change to STORE_FORM_IS_LOADING